python/ray/experimental/async_api.py

```diff
@@ -1,45 +1,10 @@
-import asyncio
-
-import ray
-from ray.experimental.async_plasma import PlasmaEventHandler
-from ray.services import logger
-
-handler = None
-
-
-def init():
-    """Initialize plasma event handlers for asyncio support."""
-    assert ray.is_initialized(), "Please call ray.init before async_api.init"
-
-    global handler
-    if handler is None:
-        worker = ray.worker.global_worker
-        loop = asyncio.get_event_loop()
-        handler = PlasmaEventHandler(loop, worker)
-        worker.core_worker.set_plasma_added_callback(handler)
-        logger.debug("AsyncPlasma Connection Created!")
-
-
 def as_future(object_id):
```

Comment:
- do we still need this? I think we can just remove the entire file right?

Reply:
Yes we can!